src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/transcripts-list-item-info/TranscriptsListItemInfo.tsx

```diff
@@ -113,6 +124,18 @@ export const TranscriptsListItemInfo = (
 
   const mainStyles = classNames(transcriptsListStyles.row, styles.listItemInfo);
   const midStyles = classNames(transcriptsListStyles.middle, styles.middle);
+  const transcriptCCDS = transcript.external_references.find((x) =>
+    x.accession_id.match('CCDS')
```

Comment:
Prefer renaming `x` to something meaningful like `item` or `xref`?

Reply:
Isn't source name a more natural place to look than accession_id?

![image](https://user-images.githubusercontent.com/6834224/125458772-7a64ad02-b84c-425f-ac21-8b41361e4400.png)

Reply:
Done